aplib/cohenstats/cohenstats.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+
+def CohenEffectSize(group1, group2):
```

Comment:
Couldn't find it, but to be honest don't see using the function very often (if at all), was set-up more for practice creating functions in the different folders.